aries-site/src/pages/_app.js

```diff
@@ -19,7 +19,7 @@ function App({ Component, pageProps }) {
 }
 
 App.propTypes = {
-  Component: PropTypes.element,
+  Component: PropTypes.func,
```

Comment:
Thanks for catching this. Saw this earlier today.

Reply:
no problem!